packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -232,6 +255,14 @@ @interface FLTMarkersController ()
 
 @implementation FLTMarkersController
 
+- (instancetype)initWithClusterManagersController:(FLTClusterManagersController *)clusterManagers
+                                          channel:(FlutterMethodChannel *)channel
+                                          mapView:(GMSMapView *)mapView
+                                        registrar:(NSObject<FlutterPluginRegistrar> *)registrar {
+  _clusterManagersController = clusterManagers;
+  return [self initWithMethodChannel:channel mapView:mapView registrar:registrar];
```

Comment:
Obj-C initializers should always delegate to the most complete initializer; the way to structure this is to have this new method become the core initializer, and the existing one should call it with a `nil` cluster manager controller.

(Both to follow standard conventions so as not to confuse people, but also because if the initializer you are calling here actually returned a different object from `[super init]` the controller set on the previous line would be lost, so this is incorrect as written.)